src/main/java/dev/austech/betterreports/utils/Common.java

```diff
@@ -30,9 +30,9 @@
 import org.bukkit.configuration.file.FileConfiguration;
 
 public final class Common {
-    private static String logPrefix = "&8[&c&l&oBetter&4&l&oReports&r&8] &f";
 
     public static void log(String message){
+        String logPrefix = "&8[&c&l&oBetter&4&l&oReports&r&8] &f";
```

Comment:
Why move this here tho?

Reply:
intelliJ was like wassup this may be final

Reply:
its only used once, it doesn't even really need to be a var

Reply:
True that

Reply:
Move to inline variable as used once